helper/build_debs.sh

```diff
@@ -26,6 +26,7 @@ Architecture: all
 Maintainer: Mondoo <hello@mondoo.com>
 Description: Mondoo Compatability Wrapper for cnspec
```

Comment:
we should rename this in something like `Mondoo meta-package for cnquery and cnspec`